src/features/NFT/NFTCardContent.tsx

```diff
@@ -0,0 +1,58 @@
+import { OpenSeaNFT } from '@services/ApiService/OpenSea';
+
+const SUPPORTED_VIDEO_EXTENSIONS = ['mov', 'mp4', 'webm', 'ogg'];
+
+export const NFTCardContent = ({ nft }: { nft: OpenSeaNFT }) => {
+  const url = nft.image_preview_url ?? nft.image_url;
```

Comment:
What's the difference between the preview and the image ?

Reply:
Preview is usually smaller size, but the API is pretty inconsistent, so we can try both if one of them isn't defined for some reason.

Reply:
Perfect then :) I didn't knew the preview could be missing :)